turbomoleio/input/define.py

```diff
@@ -699,9 +700,10 @@ def _define_symmetry(self):
         Returns:
             None
         """
+        sym_eps = self.parameters.get("sym_eps") or ""
```

Comment:
Do we leave Turbomole's default (i.e. passing "" when it is not set in the parameters) or do we set a turbomoleio default ? The "advantage" of doing it in turbomoleio is that if it changes in Turbomole, the integration tests will continue to work (and the simulations will/should continue to give the same, or at least similar, results). Most probably this is not the only parameter/place where we could have such an option. We could maybe open an issue about this and likely it should also be something to be discussed with the turbomole developers. What do you think ?

Reply:
Indeed I would not change a default value in a minor PR like this. It would also be better to keep the option to have the None value to still rely on turbomole defaults. It might be a good idea to open an issue to keep track of this.